yarn.lock

```diff
@@ -2443,9 +2490,13 @@ minimist@0.0.8:
   version "0.0.8"
   resolved "https://registry.yarnpkg.com/minimist/-/minimist-0.0.8.tgz#857fcabfc3397d2625b8228262e86aa7a011b05d"
 
-minimist@1.2.0, minimist@LiskHQ/minimist#2f27878, minimist@^1.2.0:
+minimist@1.2.0, minimist@^1.2.0:
+  version "1.2.0"
+  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.0.tgz#a35008b20f41383eec1fb914f4cd5df79a264284"
+
+minimist@LiskHQ/minimist#2f27878:
   version "1.2.0"
-  resolved "https://codeload.github.com/LiskHQ/minimist/tar.gz/2f27878dc10167bea5d0e56705b03158f7e5d876"
+  resolved "https://codeload.github.com/LiskHQ/minimist/tar.gz/2f27878"
```

Comment:
sometimes a full hash, sometimes not

Reply:
Thanks for reviewing the yarn.lock - I will have a closer look